.github/workflows/backport-comment.yml

```diff
@@ -0,0 +1,44 @@
+name: Backport Command
+
+on:
+  issue_comment:
```

Comment:
Isn't it PR comment? or are they the same? We should restrict the actions on PR to master branch.